Rakefile

```diff
@@ -1,4 +1,5 @@
 require 'open-uri'
+require 'bundler/gem_tasks'
```

Comment:
interesting, that seems to have worked! Glad you found something lower-hanging :-)

https://github.com/usertesting/biscuit/runs/2739914470?check_suite_focus=true